operator/docs/user-guides/open-telemetry.md

```diff
@@ -0,0 +1,163 @@
+    global:
+      otlp:
+        streamLabels:
+          # [...]
+        structuredMetadata:
+          resourceAttributes:
+          - name: "process.command_line"
+          - name: "k8s\\.pod\\.labels\\..+"
+            regex: true
+          scopeAttributes:
+          - name: "service.name"
+          logAttributes:
+          - name: "http.route"
+```
+
+The previous example also shows that the attribute names can be expressed as _regular expressions_ by setting `regex: true`.
+
+Using a regular expression makes sense when there are many attributes with similar names that should be mapped into Loki. It is not recommended to be used for stream labels, as it can potentially create a lot of data.
+
+### Customizing OpenShift Defaults
+
+The `openshift-logging` tenancy mode contains its own set of default attributes. Some of these attributes (called "required attributes") can not be removed by applying a custom configuration, because they are needed for other OpenShift components to function properly. Other attributes (called "recommended attributes") are provided but can be disabled in case they influence performance negatively. The complete set of attributes is documented in the [data model](rhobs-data-model) repository.
+
+Because the OpenShift attribute configuration is applied based on the tenancy mode, the simplest configuration is to just set the tenancy mode and not apply any custom attributes. This will provide instant compatibility with the other OpenShift tools.
+
+In case additional attributes are needed, either as stream labels or structured metadata, the normal custom attribute configuration mentioned above can be used. In contrast to the Grafana default labels, providing a custom attribute configuration in the `openshift-logging` tenancy mode does not remove the defaults. Instead the custom configuration is **merged** with the default configuration.
+
+#### Removing Recommended Attributes
+
+In case of issues with the default set of attributes, there is a way to slim down the default set of attributes applied to a LokiStack operating in `openshift-logging` tenancy mode:
+
+```yaml
+# [...]
+spec:
+  tenants:
+    mode: openshift-logging
+    openshift:
+      otlp:
+        disableRecommendedAttributes: true # Set this to remove recommended attributes
+```
+
+Setting `disableRecommendedAttributes: true` reduces the set of default attributes to only the "required attributes".
+
+This option is meant for situations when some of the default attributes cause performance issues during ingestion of logs or if the default set causes excessive use of storage.
+
+Because the set of required attributes only contains stream labels and even reduces that set, only setting this option will negatively affect query performance. It needs to be combined with a custom attribute configuration that reintroduces attributes that are needed under the specific circumstances, so that the data contained in those attributes is available again.
```

Comment:
```suggestion
Because the set of required attributes only contains a subset of the default stream labels, only setting this option will negatively affect query performance. It needs to be combined with a custom attribute configuration that reintroduces attributes that are needed for queries so that the data contained in those attributes is available again.
```